app/assets/javascripts/discourse/app/components/glimmer-component-with-deprecated-parent-view.js

```diff
@@ -4,16 +4,31 @@ import {
   setInternalComponentManager,
 } from "@glimmer/manager";
 import EmberGlimmerComponentManager from "@glimmer/component/-private/ember-component-manager";
+import { valueForRef } from "@glimmer/reference";
 
 class GlimmerComponentWithParentViewManager extends CustomComponentManager {
-  create(owner, componentClass, args, environment, dynamicScope) {
+  create(
+    owner,
+    componentClass,
+    args,
+    environment,
+    dynamicScope,
+    callerSelfRef
+  ) {
     const result = super.create(...arguments);
 
     result.component.parentView = dynamicScope.view;
     dynamicScope.view = result.component;
 
+    const _target = valueForRef(callerSelfRef);
+    result.component._target = _target;
+
     return result;
   }
+
+  getCapabilities() {
```

Comment:
This doesn't get tested right?

Reply:
The acceptance test of d-button relies on it. If `createCaller` is not true, then `_target` will be null and it'll be unable to trigger the action.